source/PlayerInfo.h

```diff
@@ -355,6 +359,8 @@ class PlayerInfo {
 
 
 private:
+	static PlayerInfo activePlayer;
```

Comment:
Do we normally use static variables in classes? I think usually something like this goes into the unnamed namespace in the cpp.

Reply:
The source is inconsistent on static vs. local. In this case, static makes more sense. It provides a promise that the `Player &` from ActivePlayer() will return something that'll never be deallocated.